### ISSUE TITLE
Create API review for swagger changes in PR

### ISSUE THREAD

**Issue by praveenkuttappan** [Azure/azure-sdk-tools#3600]
Once Swagger review format is finalized and supported in APIView, we need to create reviews whenever someone changes swagger file. So this step will create a baseline swagger review using swagger files in destination branch and modified api revision using source branch of the swagger PR.

This depends on #2342

**Comment by praveenkuttappan** [comment]
Duplicate of #4441
